### DISCUSSION TITLE
Stop transmission-daemon with parent?

### DISCUSSION THREAD

**Discussion by vorot93** [transmission/transmission#3774] [Q&A]
How to make transmission-daemon stop if it's launched by another process and that process dies by SIGKILL (i.e. no cleanup there)?

**Comment by fredo-47** [comment]
Use `transmission-remote --exit` in order to connect to the daemon and tell it to quit?
Maybe `killall transmission-daemon` works too?

**Reply by vorot93** [reply]
I'm using transmission-daemon in my app in lieu of embedded BitTorrent client, but for this I need its lifespan to match my app.
